README.md

```diff
@@ -171,6 +171,14 @@ Now you can connect to the mysql running in the docker container, also to connec
 10. Click the green "Run" button or "Shift + F10" to start the application
 11. After starting the Tomcat server on port `8080`, attempt to access the dummy route `http://localhost:8080/test` using the `GET` method in Postman or ThunderClient while providing the `bearer token`. If the terminal displays `test123`, it indicates that the setup has been successful.
 
+## Contributing
+### Code Formatting
+
+This repo uses https://github.com/diffplug/spotless/tree/main/plugin-maven#java for formatting files.
+Please run `mvn spotless:apply` before check-in to fix any formatting errors.
```

Comment:
check-in means?

Reply:
By check-in, I meant pushing the code in the repository. I'll make it more clear. Thank you